### ISSUE TITLE
Fix accessibility issues w/ color contrast, match other styles to conventions

### ISSUE THREAD

**Issue by robbwdoering** [ipython/ipython-website#169]
Hi, thanks for your hard work on such a great project! Have used it for years without ever visiting the website.

I can infer that the website is a labor of love (esp. via the themes at the bottom, very cute) but it screams "talented backend engineer does frontend", if you'll excuse the colloquialism. As the title states, I have ~~two~~ three concerns:

1. Body text on the splash page fails a "Contrast Check", which is not only a branding issue but a serious accessibility concern -- I had a coworker who would get bad headaches trying to read such text, for a concrete example. See tests for both [the winter theme](https://webaim.org/resources/contrastchecker/?fcolor=9CA3AF&bcolor=4588F6) and [the main (?) theme](https://webaim.org/resources/contrastchecker/?fcolor=9CA3AF&bcolor=008C93). 

2. Similarly, the pride themes are quite wholesome, but in their current state seem to be something of an accessibility nightmare for their own reasons (on top of contrast issues). I don't have a link for this one, but mostly just because there's no accessibility rules or design principles covering text that changes colors *mid glyph* 😬

3. I think there's good, objective reasons to avoid the clearly hand-rolled buttons and headers, both for maintainability and for clarity. See, for example, point 10 here: https://principles.design/examples/the-ten-principles-of-inclusive-design

**I'd be honored to submit a pull request sometime with concrete proposals if y'all want the help**, but figured I'd check the general sentiment on the issue before putting in the effort!

**Comment by Carreau** [comment]
Yes, that was crude attempt; and i'm not indeed a frontend designer. the winter theme will toggle off soon, and if the default theme is readable that would be great.

I'll merge any PR that improve the website I was just tired of a maybe decade old version with out of date informations.

And yes it is hard to have pride theme that respect a11y with pure CSS. It's more a subtle sign the the project/I care, than for actual usage.
